src/scenic/simulators/carla/actions.py

```diff
@@ -108,21 +108,21 @@ def applyTo(self, obj, sim):
 
 
 class SetAutopilotAction(VehicleAction):
+    """Enable or disable CARLA autopilot with optional Traffic Manager settings.
+
+    Arguments:
+        enabled: Enable or disable autopilot (bool)
+        kwargs: Additional autopilot options such as:
```

Comment:
The formatting of this nested list isn't working. You can add a blank line here, then asterisks at the start of each item, for it to work. Please also put the names of the options within double (not single) backticks so that they'll be formatted like code.